### ISSUE TITLE
ATT/Status

### ISSUE THREAD

**Issue by i982** [torilmud/issues#972]
not really an issue more of an idea.. 
would there be a way to make stats appear at the bottom of att? i feel like it would eliminate another command that should really just list under att for attributes.. and remove or clean up the duplicate info such as hit/dam, ac agi, bonus, saving throws, resistances.
more or less this type att and get:

Level: 50   Race: Shield Dwarf   Class: Paladin     Size: Medium
---------------------------------------------------------------------------
Unmodified Attributes:
STR: 108  AGI: 204  DEX: 110  CON: 100
POW:  40  INT:  69  WIS:  63  CHA:  42
---------------------------------------------------------------------------
Racial Attributes:
STR: 140   AGI: 198   DEX: 110   CON: 130
INT:  62   WIS:  63   CHA:  35   POW:  40
---------------------------------------------------------------------------
Hit points   :   726/895      Move points   :   180/180  
Armor Class  :  -100          Agility bonus :   -33
To-hit bonus :   +42          Damage bonus  :   +87
---------------------------------------------------------------------------
Alignment: Chaotic Evil
---------------------------------------------------------------------------
Saving Throws: PAR[-9]  ROD[]  PET[-6]  BRE[-16]  SPE[-5]
---------------------------------------------------------------------------
Magic Resistance: 22 percent
---------------------------------------------------------------------------
Damage Resistances
 Fire  :   50%  Cold  :   15%  Elect :   15%  Acid  :   15%  Poison:   10%
---------------------------------------------------------------------------
Load carried: Not a problem 
Wimpy: not set

att and stats all rolled together in a nice neat format:) just and idea. would love feedback or better ways to integrate 2  menus to one.
id even love to see score status rolled together... the bottom of the score sheet is just a non informative version of status anyways..

Name: Billybong                               Surname: Thorton                        
Name status: Pending
Title:
Experience Progress: 3941 % 
Acheron Kill Count: 0
---------------------------------------------------------------------------
Coins carried:    0 platinum     0 gold     0 silver     0 copper
Coins in bags: 8730 platinum    58 gold   114 silver     0 copper
Coins in bank: 923531 platinum  159228 gold  127949 silver     0 copper
---------------------------------------------------------------------------
Prestige: Never Going To Be As Much As Pidibeple   <- (Everyone's Prestige Should Just Say That...)
---------------------------------------------------------------------------
Citizen of:   Mithril Hall
Outcast from: Leuthilspar
Playing time: 6 days / 20 hours / 4 minutes
---------------------------------------------------------------------------
Status of Havix:  Standing
Innate Racial Bonus:
---------------------------------------------------------------------------
well endowed         summon Mount         anti-evil          turn undead
---------------------------------------------------------------------------
Enchantment Effects:
---------------------------------------------------------------------------
detect evil               detect invisible          haste                     
infravision               no summon                 protection from evil      
sense life                slow poison 
---------------------------------------------------------------------------
Spell Effects:
---------------------------------------------------------------------------
Detect Invisibility       ( 17:27) 
---------------------------------------------------------------------------

anyways just some quality of life change ideas. thanks for lookin at.

**Comment by GeneralGormal** [comment]
Yeah, even like STR: 100 (135) Dex: 100 (80) or whatever to help keep the window smaller. Better sorting/display of status and score would be nice as well.

**Comment by i982** [comment]
> Yeah, even like STR: 100 (135) Dex: 100 (80) or whatever to help keep the window smaller. Better sorting/display of status and score would be nice as well.

thats even better
